<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>国际象棋马的遍历</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>请选择起始位置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>开始演示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseBtn">
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetBtn">
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="speedBtn">
        <property name="text">
         <string>速度：中等</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu2"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>开始演示</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>重置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionStart</sender>
   <signal>triggered()</signal>
   <receiver>startBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>resetBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
